_data-prepper/trace-analytics.md

```diff
@@ -0,0 +1,381 @@
+---
+layout: default
+title: Trace analytics
+nav_order: 21
+---
+
+# Trace analytics
+
+<!---Trace analytics allows you to collect trace data and customize a pipeline that ingests and transforms the data for use in OpenSearch.---> 
+
+This page provides an overview of the trace analytics workflow in Data Prepper, how to configure it, and how to visualize trace data.
```

Comment:
Avoid 'this page'. Talk about when or why you would use the trace analytics use case.